BHoM_Engine/Compute/RecordEvent.cs

```diff
@@ -46,6 +46,35 @@ public static bool RecordEvent(string message, EventType type = EventType.Unknow
 
         /***************************************************/
 
+        [Description("Record an event with details of a C# exception within the BHoM logging system.")]
+        [Input("exception", "The C# exception being caught to provide the event and stack information for.")]
+        [Input("message", "An optional additional message which will be displayed first in the event log.")]
+        [Output("success", "True if the event has been successfully recorded as a BHoM Event.")]
+        public static bool RecordEvent(Exception exception, string message = "", EventType type = EventType.Unknown)
```

Comment:
Might as well get this input arg added I think

Reply:
Agreed, added.